src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp

```diff
@@ -435,11 +435,8 @@ real64 CompositionalMultiphaseFVM::calculateResidualNorm( real64 const & GEOS_UN
     }
     residualNorm = sqrt( globalResidualNorm[0] * globalResidualNorm[0] + globalResidualNorm[1] * globalResidualNorm[1] );
 
-    if( getLogLevel() >= 1 && logger::internal::rank == 0 )
-    {
-      std::cout << GEOS_FMT( "        ( R{} ) = ( {:4.2e} )        ( Renergy ) = ( {:4.2e} )",
-                             coupledSolverAttributePrefix(), globalResidualNorm[0], globalResidualNorm[1] );
-    }
+    GEOS_LOG_LEVEL_INFO_RANK_0( logInfo::ResidualNorm, GEOS_FMT( "        ( R{} ) = ( {:4.2e} )        ( Renergy ) = ( {:4.2e} )",
```

Comment:
`cout` was used here for a reason: all those GEOS_LOG macros add `endl` at the end, but residuals from solvers are all printed in a line, otherwise it will look very long and ugly
please revert or find a way to preserve the current form for residual output

Reply:
@arng40 maybe you can add a different macro that does not break the line?